### ISSUE TITLE
Publish arm Tooljet docker images

### ISSUE THREAD

**Issue by bfosberry** [ToolJet/ToolJet#7222]
### Is your proposal related to a problem?

Currently running the published image on an arm system results in UI errors. It is difficult to determine if the cause of this issue is the platform, or some other image incompatibility, however Tooljet is essentially unusable for me on arm6t4 using the commands and browsers provided by Tooljet engineers.

### Describe the solution you'd like

Support for arm64, specifically
* Supporting amongst key plugins needed to run Tooljet
* Dockerfile support for arm64 within the repo
* Published linux/arm64 images on dockerhub so testing and running self-hosted is simpler

### Describe alternatives you've considered

I've attempted to run Tooljet cross-arch and have been unable to, even with support from Adish in slack (https://tooljet.slack.com/archives/C01TY0ZGH6X/p1690950059289199)

### If the feature is approved, would you be willing to submit a PR?
No

**Comment by austinbspencer** [comment]
I am also hoping to see this and surprised it isn't already available.

**Comment by aliaksandr-biarozka** [comment]
I'm also interested in it. Are the any plans for it?

**Comment by davyzhang** [comment]
rikmeijer84/tooljet:latest

which this can help somebody, I didn't build this, just happen to find this

**Comment by abulka** [comment]
I tried this image
```
% docker run \
  --name tooljet \
  --restart unless-stopped \
  -p 8099:80 \
  -v tooljet_data:/var/lib/postgresql/13/main \
  rikmeijer84/tooljet:latest
```
which downloaded ok, but got error `Error: you need to provide a host and port to test.` which seems to be related to #5749. Perhaps a later image is needed to get around this but I don't know how to contact  rikmeijer84. 

Interesting that #6195 seems to use this image successfully - any tips appreciated.

**Comment by davyzhang** [comment]
> I tried this image
> 
> ```
> % docker run \
>   --name tooljet \
>   --restart unless-stopped \
>   -p 8099:80 \
>   -v tooljet_data:/var/lib/postgresql/13/main \
>   rikmeijer84/tooljet:latest
> ```
> 
> which downloaded ok, but got error `Error: you need to provide a host and port to test.` which seems to be related to #5749. Perhaps a later image is needed to get around this but I don't know how to contact rikmeijer84.
> 
> Interesting that #6195 seems to use this image successfully - any tips appreciated.

Sorry for the confusion. I used another approach. I started with official setup instructions here : https://docs.tooljet.com/docs/setup/docker

and then change the image from 
      #image: tooljet/tooljet:latest
to
    image: rikmeijer84/tooljet:latest

It worked on my arm server

**Comment by nicola** [comment]
`postgrest (terminated by SIGKILL)`

:(

**Comment by abulka** [comment]
As the `image: rikmeijer84/tooljet:latest` mentioned above is over a year old now, I searched dockerhub and found something a bit later (Aug 2024)
```
image: rockv3/tooljet:lts-2.50.19
```
which ran OK on my Mac M2 docker (following the same instructions as @davyzhang mentioned).  

However its not the latest Version 3 series. 

I also found `rockv3/tooljet:v2.66.1` which is a tiny bit more recent, but couldn't get that to run.

**Comment by laustindasauce** [comment]
@abulka your Mac is running amd64, not arm. So you should be able to use the official tooljet Docker image following [their guide for Docker](https://docs.tooljet.com/docs/setup/try-tooljet/) or with [Docker Compose](https://docs.tooljet.com/docs/setup/docker/). This issue is specifically for support with arm based operating systems.

**Comment by abulka** [comment]
> @abulka your Mac is running amd64, not arm. So you should be able to use the official tooljet Docker image following [their guide for Docker](https://docs.tooljet.com/docs/setup/try-tooljet/) or with [Docker Compose](https://docs.tooljet.com/docs/setup/docker/). This issue is specifically for support with arm based operating systems.

Not true. My Mac M2 is running on Apple Silicon and is ARM64 not AMD64. All the latest Macs (since Nov 2020) are arm.

**Comment by laustindasauce** [comment]
I stand corrected. Apologies, I didn’t realize Apples chips switched to ARM based. Have you tried Rosetta Virtualization to run the official image?
